models/staging/base_parta_diagnosis_code.sql

```diff
@@ -0,0 +1,53 @@
+with diagnosis as (
```

Comment:
Had to add base models to use the pivot macro from dbt utils. The reference had to be a source or model. Could not use var syntax.